core/arch/arm/plat-nuvoton/platform_config.h

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2014-2023, Linaro Limited
+ * Copyright (C) 2022-2023 Nuvoton Ltd.
+ */
+
+#ifndef PLATFORM_CONFIG_H
+#define PLATFORM_CONFIG_H
+
+#include <mm/generic_ram_layout.h>
+
+/* Make stacks aligned to data cache line length */
+#define STACK_ALIGNMENT		64
+#define DRAM0_BASE	0x00000000
```

Comment:
Please indent all these values so they align with `64` above and `UART0_BASE` below.

Reply:
Done